integration_tests/src/main/python/data_gen.py

```diff
@@ -583,20 +583,13 @@ class TimestampGen(DataGen):
     """Generate Timestamps in a given range. All timezones are UTC by default."""
```

Comment:
nit: Could you file a follow on issue to look at how we might be able to work around the range limitations on input to timestamps and dates? It might be nice to inject some processing or something to make it so that we can do the full range that spark supports instead of just what python can handle.

Reply:
So to clarify, would this mean supporting negative timestamps that Spark can handle since the underlying value is microseconds? Or is there something missing here from the `0001-01-01` to `9999-12-31` range?

Reply:
Yes it would be nice to handle generating negative timestamp values, and positive values that support the full range that Spark supports. Up to 6 digit years. I think this is minor, but it would nice to have an issue to look into it.

Reply:
https://github.com/NVIDIA/spark-rapids/issues/10040